pkg/action/all.go

```diff
@@ -12,6 +13,16 @@ import (
 func All(cluster string) ([]string, error) {
 	var actions []string
 	{
+		_, err := os.Stat(fmt.Sprintf("cmd/%s", cluster))
+		if os.IsNotExist(err) {
+			err = os.Mkdir(fmt.Sprintf("cmd/%s", cluster), os.ModePerm)
+			if err != nil {
+				return nil, microerror.Mask(err)
+			}
```

Comment:
`All` is only a reading function. Writing on the file system is somehow not so cool when you just want to fetch information. So in case the directory does not exist we should maybe just return an empty list.